core/tests/protractor_utils/ExplorationEditorHistoryTab.js

```diff
@@ -100,7 +101,8 @@ var ExplorationEditorHistoryTab = function() {
           closeStateHistoryButton,
           'Close State History button is not clickable');
         expect(await closeStateHistoryButton.isDisplayed()).toBe(true);
```

Comment:
The action.click method waits for the button to be present before clicking on it. So we can remove this line.

Reply:
Ok I will remove it.